docs/content/dev/config.md

```diff
@@ -0,0 +1,74 @@
+# Config
+
+The configuration of HedgeDoc is handled entirely by environment variables. Most of these variables are prefixed with `HD_` (for HedgeDoc).
+
+## Configuring HedgeDoc for development
+
+NestJS - the framework we use - is reading the variables from the environment and also from the `.env` file in the root of the project. To make it easy to get the project started we included an example `.env` file in the root of this project called `.env.example`.
+
+## How the config code works
+
+The config of HedgeDoc is split up into **six** different modules:
+
+`app.config.ts`
+: General configuration of the app
+
+`auth.config.ts`
+: Which authentication providers are available and which options are set
+
+`csp.config.ts`
+: Configuration for [Content Security Policy][csp]
+
+`database.config.ts`
+: Which database should be used
+
+`hsts.config.ts`
+: Configuration for [HTTP Strict-Transport-Security][hsts]
+
+`media.config.ts`
+: Where media files are being stored
+
+Each of those files (except `auth.config.ts` which is discussed later) consists of three parts:
+
+1. An Interface
+2. A Joi schema
+3. A default export
+
+### Interface
+
+The interface just describes what options the configuration has and how the rest of HedgeDoc can use them. All enums that are used in here are put in their own files with the extension `.enum.ts`.
+
+### Joi Schema
+
+We use [Joi][joi] to validate each provided configuration to make sure the configuration of the user is sound and provide helpful error messages if that's not the case.
```

Comment:
```suggestion
We use [Joi][joi] to validate each provided configuration to make sure the configuration of the user is sound and provides helpful error messages otherwise.
```

Reply:
You're sure about the _provides_? Maybe I'm just overlooking something, but the s doesn't seem right there for me.

The _otherwise_ is definitely a better choice.

Reply:
I think provides is correct.

Reply:
btw. done